es-rest-client-1.3/core/src/main/java/org/elasticsearch/search/aggregations/metrics/stats/StatsHelper.java

```diff
@@ -11,10 +10,10 @@
 
 public class StatsHelper {
     public static InternalAggregation fromXContent(final String name, final Map<String, Object> map, final AggregationsManifest manifest) {
-        final long count = nodeLongValue(map.get("count"));
-        final double sum = nodeDoubleValue(map.get("sum"));
-        final double min = nodeDoubleValue(map.get("min"));
-        final double max = nodeDoubleValue(map.get("max"));
+        final long count = (map.get("count") == null ? 0L : nodeLongValue(map.get("count")));
+        final double sum = (map.get("sum") == null ? 0 : nodeDoubleValue(map.get("sum")));
+        final double min = (map.get("min") == null ? 0 : nodeDoubleValue(map.get("min")));
+        final double max = (map.get("max") == null ? 0 : nodeDoubleValue(map.get("max")));
```

Comment:
on second thought, I'm going to change this to check for count == 0 instead, just like doXContentBody.